local-cli/generator/index.js

```diff
@@ -8,11 +8,41 @@
  */
 'use strict';
 
+var execSync = require('child_process').execSync;
 var fs = require('fs');
 var path = require('path');
+var semver = require('semver')
```

Comment:
I don't have too strong feelings about it but we should be conservative about adding dependencies to react-native-cli. It should be shallow to not bother people IMHO